ember-cli-build.js

```diff
@@ -3,7 +3,9 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
-    // Add options here
+    babel: {
+      includePolyfill: true
```

Comment:
In order to ensure that this library works without the babel polyfill, I think we should not add it...